packages/sanity/src/desk/panes/documentList/DocumentListPaneContent.tsx

```diff
@@ -224,7 +226,7 @@ export function DocumentListPaneContent(props: DocumentListPaneContentProps) {
         <CommandListBox>
           <CommandList
             activeItemDataAttr="data-hovered"
-            ariaLabel="Document list"
+            ariaLabel={`List of ${paneTitle} documents`}
```

Comment:
Is `paneTitle` the right name here?

I'm reading this as a way that I can specify the full pane title, when in reality I can only specify a portion of it. So if I pass, let's say _All published documents_, it will read _List of All published documents documents_. I suggest either renaming the property or making the default `List of ${documentType} documents`